src/utils.js

```diff
@@ -0,0 +1,65 @@
+/* jshint -W041*/ // ignore == to compare null warning
+// var _ = require('lodash');
```

Comment:
We probably will need `lodash` soon, but if we don't need it right now, maybe just remove this line?